src/java_bytecode/Makefile

```diff
@@ -36,10 +36,53 @@ INCLUDES= -I ..
 include ../config.inc
 include ../common
 
-CLEANFILES = java_bytecode$(LIBEXT)
+
+SOURCE_DIR := library/src
+BINARY_DIR := library/classes
+CPROVER_SRC_DIR := library/src/org/cprover
+
+FIND := find
+
+JAVAC := javac
+JFLAGS := -sourcepath $(SOURCE_DIR) -d $(BINARY_DIR) -XDignore.symbol.file
+
+CLASSPATH := SOURCE_DIR
+
+all_javas := $(wildcard $(CPROVER_SRC_DIR)/*.java) #$(shell find $(SOURCE_DIR) -name '*.java')
```

Comment:
Change to `$(wildcard library/*.java library/*/*.java library/*/*/*.java library/*/*/*/*.java)`.
Variables in a `Makefile` are always capital letters.

Reply:
Done.